### ISSUE TITLE
`/irp_checkpoint.irp.o` Missing

### ISSUE THREAD

**Issue by TApplencourt** [LCPQ/quantum_package#55]
Witch the new IRPF90 version (1.6.1), the EZFIO folder in QP don't compile. Throw this error :
```bash
`bash -c "[[ -e ../libezfio.a ]] && rm ../lib/libezfio.a"
make[1]: [../lib/libezfio.a] Error 1 (ignored)
ar cru ../lib/libezfio.a IRPF90_temp/irp_stack.irp.o  IRPF90_temp/libezfio_error.irp.o IRPF90_temp/libezfio_error.irp.module.o IRPF90_temp/libezfio_file.irp.o IRPF90_temp/libezfio_file.irp.module.o IRPF90_temp/libezfio_util.irp.o IRPF90_temp/libezfio_util.irp.module.o IRPF90_temp/libezfio_groups.irp.o IRPF90_temp/libezfio_groups.irp.module.o  IRPF90_temp/irp_touches.irp.o IRPF90_temp/irp_checkpoint.irp.o 
ar: IRPF90_temp/irp_checkpoint.irp.o: No such file or directory
make[1]: *** [../lib/libezfio.a] Error 1
make[1]: Leaving directory `/tmp/quantum_package/EZFIO/src'
make: *** [default] Error 2
```

HOT FIX
=======

In `${QPACKAGE_ROOT}/scripts/install_irpf90.sh` change 
```
URL="https://github.com/LCPQ/${BASE}/archive/master.tar.gz"
``` 
into
 ```
URL="https://github.com/TApplencourt/${BASE}/archive/master.tar.gz"
```

Delete `${QPACKAGE_ROOT}/irp90`, and run `setup_environement.sh`

**Comment by TApplencourt** [comment]
@scemama  it's related to #49 ?

**Comment by scemama** [comment]
No.

**Comment by scemama** [comment]
Fixed with IRPF90 v 1.6.2
